scripts/handler.js

```diff
@@ -0,0 +1,34 @@
+const schedule = require('node-schedule');
+const db = require('../app/database/dbConfig.js');
+const jobsModel = require('../app/models/jobsModel.js');
+const mailer = require('../modules/mailer.js');
+
+/**
+ * Adds scheduled jobs for sending emails by checking for
+ * classes starting in 30 min every 15 min using chron job 
+ * and jobsModel script. Loop through each class to get 
+ * clients' emails using knex join function. Finally, 
+ * send the email to all clients of each class using the 
+ * mailer script.
+ */
+async function scheduleEmail() {
+    schedule.scheduleJob('*/15 * * * *', async function() {
+        const classes = await jobsModel.getByMinutesFromNow(30);
+        classes.forEach(async function(classInfo) {
+            const rows = await db('clients')
```

Comment:
This should be moved into the clients model. We want all knex/database related code to be in the models folder for consistency
Since you are making a join from clients to the jobs table, you should put this function in the models folder. You can name this something like: `getEmailByJoinJobs`

Reply:
Moved